java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3473,6 +3473,16 @@ public final ColumnVector listSortRows(boolean isDescending, boolean isNullSmall
     return new ColumnVector(listSortRows(getNativeView(), isDescending, isNullSmallest));
   }
 
+  /**
+   * Generate list offsets from sizes of each list.
+   * NOTICE: This column should type in INT32. And no null and negative value is allowed.
```

Comment:
Is this requirement for the input column ? What if it is not matched ? Better to add comment for it.

Reply:
Fixed